cmd/kyma/install/cmd.go

```diff
@@ -100,13 +100,14 @@ The standard installation uses the minimal configuration. The system performs th
 	cobraCmd.Flags().StringVarP(&o.Source, "source", "s", DefaultKymaVersion, `Installation source. 
 	- To use the specific release, write "kyma install --source=1.3.0".
 	- To use the latest master, write "kyma install --source=latest".
+	- To use the latest published master, write "kyma install --source=latest-published".
```

Comment:
Here it might not be clear what "latest published" means. Maybe we should detail what will happen.
```suggestion
	- To use the latest published master, which is the latest commit with released images, write "kyma install --source=latest-published".
```